src/client/components/intro/Intro.js

```diff
@@ -30,6 +30,26 @@ const Intro = () => {
     }
   };
 
+  const greeting = () => {
+    const words = [
```

Comment:
Move this array outside of the function to the top of the file so it isn't being reconstructed every time the function executes